app/src/containers/ConnectedSidebar/tests/index.test.tsx

```diff
@@ -63,17 +69,29 @@ describe('components/ConnectedSidebar', () => {
         </MemoryRouter>
       </Provider>
     );
+
+    expect(wrapper.find(`Menu`).last().prop('children')).toMatchSnapshot();
+  });
+
+  it('correctly expand users menu', () => {
+    const wrapper = mount(
+      <Provider store={store}>
+        <MemoryRouter initialEntries={[{ pathname: `/admin`, hash: '', search: '', state: {} }]}>
+          <ConnectedSidebar />
+        </MemoryRouter>
+      </Provider>
+    );
     expect(wrapper.find('Menu').at(0).prop('children')).toMatchSnapshot();
     wrapper.unmount();
   });
 
-  it('displays menu links for enabled Location module', () => {
+  it('correctly expand location menu', () => {
```

Comment:
@ashfahan I thought we were to remove this test cases since its already catered for in  `Test order of menu'` test case?

Reply:
these expansions tests were originally written by @HaseemUlHaq @haseemVD i just replicate them over other sidebar to like fill the gap

@HaseemUlHaq can you guide us better on this ?